nrfs/src/internal/nrfs_dispatcher.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <internal/nrfs_dispatcher.h>
+#include <internal/nrfs_callbacks.h>
+#include <internal/services/nrfs_generic.h>
+
+static const nrfs_service_cb_t services_callbacks[] = {
+#ifdef CONFIG_TEMP_SERVICE_ENABLED
```

Comment:
Symbols defined by Zephyr, based on Kconfig options, should not be referred directly from this repository.

Reply:
done